source/BNFC.cabal

```diff
@@ -48,7 +48,7 @@ Executable bnfc
     array,
     process,
     containers,
-    pretty >=1.1 && <1.2,
```

Comment:
I think that's the case only if stack is the only way to build the project. I'm not sure I want to go that far for different reasons: stack is still relatively new ; I'm not sure if it works well for package maintainer in gnu/linux distributions (they may have different versions of the dependencies) ; I think even the stack documentation recommends that, if you plan to upload your package to hackage, you should make sure it also builds with just cabal.